index.html

```diff
@@ -90,8 +90,8 @@
             }
         }
         );
-        console.log(did.toString());
-        console.assert(did.toString() === 'did:iden3:polygon:mumbai:wzokvZ6kMoocKJuSbftdZxTD6qvayGpJb3m4FVXth');
+        console.log(did.string());
```

Comment:
My appointment. Is not correct to change `toString` to `string`. Since `toString` is primary JS interface for converting something to a string.
I can provide the same example with golang. Golang has an internal interface `Stringer`
```
type Stringer interface {
   String()
}
```
And all users try to implement this interface to convert custom objects to string. They can implement `ToString` instead of `String` and this will work OK, but for developers it will not "Go style". 

Same with JS. The developer has a habit call `toString` to convert something to `string`. It can be confusing for somebody.

`ToString` can return `[object Object]` but we can configure TS config that will validate this case.

Reply:
the explicit string is ok. To avoid possible `[object Object] `